GameData/RealismOverhaul/Engine_Configs/BE-4_Config.cfg

```diff
@@ -8,15 +8,15 @@
 //	Vulcan, New Glenn
 //
 //	Dry Mass: 2250 Kg
-//	Thrust (SL): 2400 kN
-//	Thrust (Vac): 2647.5 kN
-//	ISP: 305 SL / 341 Vac
+//	Thrust (SL): 2446.55 kN
+//	Thrust (Vac): 2714.15 kN
+//	ISP: 350 SL / 355 Vac
```

Comment:
I've seen this pairing of numbers questioned in a few places.  It is very different from the previous values.  I'd like some additional comments on it [from you or others].